src/engine/opengl.rs

```diff
@@ -29,7 +29,6 @@ pub struct OpenGlApp {
 }
 
 impl OpenGlApp {
-    #[allow(unsafe_code)]
```

Comment:
Good catch. I think I started with local allow unsafe_code and then as the functions grew I set it at the top level. So I'm fine with removing all these.